src/qt/recentrequeststablemodel.cpp

```diff
@@ -17,18 +16,18 @@
 
 #include <utility>
 
-RecentRequestsTableModel::RecentRequestsTableModel(WalletModel *parent) :
-    QAbstractTableModel(parent), walletModel(parent)
+RecentRequestsTableModel::RecentRequestsTableModel(interfaces::Wallet& wallet, OptionsModel* options_model, QObject* parent) :
```

Comment:
In commit "gui: Drop RecentRequestsTableModel dependency to WalletModel" (8aa9b53e9a47810d3ebe90743123b130cc4493c3)

Would be nice to make `options_model` and `m_options_model` references instead of pointers if they are not allowed to be null

Reply:
Sure, I think its unrelated to this PR though. For instance, the check in L126 could be removed. Lots of other members could be references including cases like `WalletView::walletModel` where there's a setter.